legal-api/src/legal_api/models/business.py

```diff
@@ -94,6 +95,20 @@ class LegalTypes(Enum):
         MISC_FIRM = 'MF'
         FINANCIAL = 'FI'
         CONT_IN_SOCIETY = 'CS'
+        # UNLIMITED_LIABILITY_COMPANY = 'UL'
+        # SOLE_PROPRIETORSHIP = 'FR'
+        # GENERAL_PARTNERSHIP = 'GP'
+        # DOING_BUSINESS_AS = 'DBA'
+        # LIMITED_PARTNERSHIP = 'LP'
+        # LIMITED_LIABILITY_PARTNERSHIP = 'LL'
+        # COMMUNITY_CONTRIBUTION_COMPANY = 'CC'
+        # XPRO_CORPORATION = 'XCR'
+        # XPRO_UNLIMITED_LIABILITY_COMPANY = 'XUL'
+        # XPRO_LIMITED_LIABILITY_COMPANY = 'RLC'
+        # XPRO_LIMITED_PARTNERSHIP = 'XLP'
+        # XPRO_LIMITED_LIABILITY_PARTNERSHIP = 'XLL'
+        # XPRO_COOPERATIVE = 'XCP'
+        # FIRM = 'FIRM'
```

Comment:
Note that these are not sorted, but their order is the same as in their original project (ie, to make it easy to compare code between projects).